src/modules/hip/kernel/pre_emphasis_filter.hpp

```diff
@@ -0,0 +1,74 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+__device__ __forceinline__ void pre_emphasis_filter_hip_compute(d_float8 *src1_f8, d_float8 *src2_f8, d_float8 *dst_f8, float4 *coeff_f4)
+{
+    dst_f8->f4[0] = src1_f8->f4[0] - (*coeff_f4 * src2_f8->f4[0]);
+    dst_f8->f4[1] = src1_f8->f4[1] - (*coeff_f4 * src2_f8->f4[1]);
+}
+
+__global__ void pre_emphasis_filter_tensor(float *srcPtr,
+                                           uint srcStride,
+                                           float *dstPtr,
+                                           uint dstStride,
+                                           float *coeffTensor,
+                                           int *srcLengthTensor,
+                                           RpptAudioBorderType borderType)
+{
+    int id_x = (hipBlockIdx_x * hipBlockDim_x + hipThreadIdx_x) * 8;
+    int id_z = hipBlockIdx_z * hipBlockDim_z + hipThreadIdx_z;
+
+    if (id_x >= srcLengthTensor[id_z])
+        return;
+
+    uint srcIdx = (id_z * srcStride) + id_x;
+    uint dstIdx = (id_z * dstStride) + id_x;
+    float coeff = coeffTensor[id_z];
+
+    d_float8 src1_f8, src2_f8, dst_f8;
+    float4 coeff_f4 =  static_cast<float4>(coeff);
+
+    rpp_hip_load8_and_unpack_to_float8(srcPtr + srcIdx, &src1_f8);
+
+    if(id_x == 0)
```

Comment:
Please remove empty blank lines at L30 and L32

Reply:
Done